app/imports/components/Projects/SingleProjectSidebar/SingleProjectSidebar.jsx

```diff
@@ -56,19 +58,47 @@ const renderSprintLinks = (
 const renderSprintListItems = (
     sprints,
     goToAddSprint,
+    currentSprintName,
     goToSprint,
     projectId,
     showAddSprint = false,
 ) => {
-    const listSprints = sprints.map(sprint => (
+    const openSprints = sprints.filter(sprint => !sprint.closed);
+    const closedSprints = sprints.filter(sprint => sprint.closed);
+
+    const listSprints = openSprints.map(sprint => (
         <ListItem
             leftIcon={sprint.closed ? <Lock /> : <DirectionsRun />}
             key={sprint._id}
-            primaryText={sprint.name}
+            primaryText={sprint.name === currentSprintName ? <b>{sprint.name}</b> : sprint.name}
```

Comment:
`sprint.name` isn't unique. Use sprint id or make sure that sprint name will be unique per project.